funding/manager.go

```diff
@@ -3126,6 +3126,15 @@ func (f *Manager) addToRouterGraph(completeChan *channeldb.OpenChannel,
 			"announcement: %v", err)
 	}
 
+	// We make sure that we set the dont_forward bit in case the peer alias
+	// is selected which means that this channel is not (yet) announced to
+	// the broader network and the flag was not set before.
+	if peerAlias != nil &&
+		!ann.chanUpdateAnn.MessageFlags.HasDontForward() {
```

Comment:
Curious to hear why you don't just set the flag if it's an unannounced channel but first check if the flag has already been set? Would look more concise without that check.